static/src/javascripts/bootstraps/enhanced/common.js

```diff
@@ -192,10 +192,8 @@ define([
                 if (config.page.contentType !== 'Network Front') {
                     history.logSummary(config.page);
                 }
-
-                if (config.page.contentType === 'Video') {
-                    history.logHistory(config.page);
-                }
+                
+                history.logHistory(config.page);
```

Comment:
Will this have any knock-on effect on video-related code (e.g. if it assumes that everything in the history is video)?